client/my-sites/plan-features/index.jsx

```diff
@@ -412,10 +418,26 @@ class PlanFeatures extends Component {
 				features,
 				planName
 			} = properties;
+			
+			let planFeatures = features;
```

Comment:
If you mean to leave "features" untouched here this won't accomplish it. If you wanted to create a new array while keeping features untouched you would need to declare it where you set the value from the map method below.